frontend/public/components/utils/list-dropdown.jsx

```diff
@@ -196,6 +196,7 @@ ListDropdown.propTypes = {
     PropTypes.shape({
       kind: PropTypes.string.isRequired,
       namespace: PropTypes.string,
+      selector: PropTypes.string,
```

Comment:
This type conflicts with the Firehose type
```suggestion
      selector: PropTypes.object,
```